.github/actions/apt-x64/action.yml

```
@@ -0,0 +1,66 @@
+name: apt
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: |
+        set -e
+        sudo apt install bison \
```

```suggestion
        sudo apt-get update
        sudo apt-get install -y \
          bison \
```
bison is easier to miss if it's part of the `apt install` line. I'd also recommend `apt-get` over `apt` for programmatic use and an explicit `apt-get update`, because from my experience the cached package lists are sometimes outdated and refer to versions that are no longer available on the package servers (because they've been superseded by a newer version).